sphinx/conf.py

```diff
@@ -123,8 +123,11 @@
 lists_root = 'http://lists.openmicroscopy.org.uk'
 downloads_root = 'https://downloads.openmicroscopy.org'
 docs_root = 'https://docs.openmicroscopy.org'
+imagesc_root = 'https://forum.image.sc'
```

Comment:
Thoughts about pointing directly at https://forum.image.sc/tags/bio-formats in this technical documentation